purchase_requisition_from_rfq/models/purchase.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    @api.multi
+    def write(self, vals):
+        for rec in self:
+            super(PurchaseOrder, rec).write(vals)
```

Comment:
- Why do we need this? If it is really necessary, please give some helpful description to reviewer.
- Not good to do super() in side for loop (please give some comments, I forgot why you need this).

Reply:
I used hook.